src/DataTypes/LearnerReport.php

```diff
@@ -104,12 +104,12 @@ class LearnerReport {
     /**
      * The UTC date the enrollment was created.
      */
-    protected DateTimeInterface $createdDate;
+    protected ?DateTimeInterface $createdDate;
 
     /**
      * The UTC date the enrollment was last updated.
      */
-    protected DateTimeInterface $modifiedDate;
+    protected ?DateTimeInterface $modifiedDate;
```

Comment:
You need to provide a default value since this is a strongly typed property
```suggestion
    protected ?DateTimeInterface $modifiedDate = null;
```